test/hotspot/jtreg/serviceability/jvmti/HiddenClass/libHiddenClassSigTest.cpp

```diff
@@ -313,11 +330,19 @@ Agent_OnLoad(JavaVM *jvm, char *options, void *reserved) {
     return JNI_ERR;
   }
 
+  err = jvmti->CreateRawMonitor("Event Monitor", &event_mon);
```

Comment:
Sorry, forgot to mention.
There is a function ' create_raw_monitor(jvmti, name)" in jvmti commo that saves a few more lines:
use
event_mon =  create_raw_monitor(jvmti, "Event Montori");

Reply:
I saw the function, but don't see much sense in it.
On error it just return null (so need to check result anyway) and it's not possible to get jvmti error

Reply:
Strange, this function should be improved to at least log the probleml.